liwords-ui/src/gameroom/board_panel.tsx

```diff
@@ -493,6 +501,7 @@ export const BoardPanel = React.memo((props: Props) => {
       if (handlerReturn === null) {
         return;
       }
+      evt.preventDefault();
```

Comment:
could we just put one blanket `evt.preventDefault()`  around line 435?

Reply:
wouldn't be a good user experience to unnecessarily block PgDn or Tab (for example)